bin/github-default-branch

```diff
@@ -153,7 +153,15 @@
       console.log(`✏️  Updating default branch to [${target}] in [${repo}]`);
     }
 
-    if (!isDryRun) {
+    const {
+      data: { default_branch },
```

Comment:
```suggestion
      data: { default_branch: defaultBranch },
```
does this work? and then remove line 162

Reply:
ya! thanks, i didn't know how to do that correctly